### PR TITLE
Fix glibc version test

### LINKED ISSUE #70: intel QAT icp_adf_userProcessToStart failed
nginx 1.10.3 +QAT_Engine-0.5.32 +QAT1.7 driver；run  the command and here is the outputs:
[root@194 conf]# /usr/local/ssl/bin/openssl  engine -t -c -vvvv qat
[error] icp_sal_userStart() - : icp_adf_userProcessToStart failed

(qat) Reference implementation of QAT crypto engine
 [RSA, DSA, DH, AES-128-CBC-HMAC-SHA1, AES-128-CBC-HMAC-SHA256, AES-256-CBC-HMAC-SHA1, AES-256-CBC-HMAC-SHA256, TLS1-PRF]
[error] icp_sal_userStart() - : icp_adf_userProcessToStart failed

     [ unavailable ]
     ENABLE_EXTERNAL_POLLING: Enables the external polling interface to the engine.
          (input flags): NO_INPUT
     POLL: Polls the engine for any completed requests
          (input flags): NO_INPUT
     SET_INSTANCE_FOR_THREAD: Set instance to be used by this thread
          (input flags): NUMERIC
     GET_NUM_OP_RETRIES: Get number of retries
          (input flags): NO_INPUT
     SET_MAX_RETRY_COUNT: Set maximum retry count
          (input flags): NUMERIC
     SET_INTERNAL_POLL_INTERVAL: Set internal polling interval
          (input flags): NUMERIC
     GET_EXTERNAL_POLLING_FD: Returns non blocking fd for crypto engine
          (input flags): NO_INPUT
     ENABLE_EVENT_DRIVEN_POLLING_MODE: Set event driven polling mode
          (input flags): NO_INPUT
     GET_NUM_CRYPTO_INSTANCES: Get the number of crypto instances
          (input flags): NO_INPUT
     DISABLE_EVENT_DRIVEN_POLLING_MODE: Unset event driven polling mode
          (input flags): NO_INPUT
     SET_EPOLL_TIMEOUT: Set epoll_wait timeout
          (input flags): NUMERIC
     SET_CRYPTO_SMALL_PACKET_OFFLOAD_THRESHOLD: Set QAT small packet threshold
          (input flags): STRING
     ENABLE_INLINE_POLLING: Enables the inline polling mode.
          (input flags): NO_INPUT

### LINKED ISSUE #202: glibc configure test not working
I'm building QAT Engine on Debian 10, which uses glibc 2.28:

```
I have no name!@6dc24c3dedf4:/$ ldd --version | head -1
ldd (Debian GLIBC 2.28-10) 2.28
```

However, when building, I can see this message in the configure log:

```
configure: glibc version < 2.23. glibc AVX-512 optimisation not supported.
```

After some investigation, I've found that the line that tests for glibc in the configure script is not working as intended. [This is the code from the configure.ac script](https://github.com/intel/QAT_Engine/blob/d9950465ae949d1570581de3ca8a2e0072eaee5d/configure.ac#L763):

```
    if test "`ldd --version | head -1 | awk '{print $4>=2.23?"1":"0"}' 2>/dev/null`" = "1"
    then
      AC_MSG_NOTICE([glibc version >= 2.23. glibc AVX-512 optimisation supported.])
    else
      AC_MSG_NOTICE([glibc version < 2.23. glibc AVX-512 optimisation not supported.])
    fi
```

If I manually run that command on my machine, I get a syntax error:

```
I have no name!@a21b2db63f19:/$ ldd --version | head -1 | awk '{print $4>=2.23?"1":"0"}'
awk: line 1: syntax error at or near =
```

It looks like `awk` is missing some parenthesis around the condition:

```
I have no name!@a21b2db63f19:/$ ldd --version | head -1 | awk '{print ($4>=2.23)?"1":"0"}'
1
```

```
I have no name!@a21b2db63f19:/$ ldd --version | head -1 | awk '{print ($4>=2.29)?"1":"0"}'
0
```

### PR DESCRIPTION
**Pull request by alemorcuq**
Fixes the glibc version test in the configure.ac script as detailed in #202.

- fixes #202

### CODE REVIEW

**Comment by Yogaraj-Alamenda** [comment]
Thanks for the Contribution. The Patch will go through the internal process and will be released in upcoming release.
Could you please sign the commit message ?

 Signed-off-by: "Full name" <Email ID>

**Comment by alemorcuq** [comment]
Done 👍

File: configure.ac [hunk 1]
```diff
@@ -760,13 +760,13 @@ linux*)
   AC_CHECK_PROG(USE_HEAD, head, yes, no)
   if test $USE_LDD == "yes" -a $USE_HEAD
   then
-    if test "`ldd --version | head -1 | awk '{print $4>=2.23?"1":"0"}' 2>/dev/null`" = "1"
+    if test "`ldd --version | head -1 | awk '{print ($4>=2.23)?"1":"0"}' 2>/dev/null`" = "1"
     then
       AC_MSG_NOTICE([glibc version >= 2.23. glibc AVX-512 optimisation supported.])
```
Comment (Yogaraj-Alamenda) [inline configure.ac:765]: This came to our internal review and many users misunderstand about the AVX512 multibuffer optimization. So it is better we add it for glibc AVX512 memcpy optimization in this message.

Can we add that as part of this  commit  ? and include it in the upcoming release ?

**Comment by Yogaraj-Alamenda** [comment]
Changes merged and available v0.6.13 release

**Pull request closed without merging**